bundles/test/org.eclipse.smarthome.magic/ESH-INF/thing/channel-types.xml

```diff
@@ -51,13 +51,11 @@
 		<item-type>String</item-type>
 		<label>Alert</label>
 		<description>The alert channel allows a temporary change to the bulb’s state.</description>
-		<state>
```

Comment:
This breaks backward compatibility as it removes the state options, which are still a valid information.